Makefile

```
@@ -0,0 +1,80 @@
+# ==============================================================================
+# VARIABLES
+
+# -- Hugo
+THEME = blowfish
+
+# -- Docker
+# Get the current user ID to use for docker run and docker exec commands
+DOCKER_UID = $(shell id -u)
+DOCKER_GID = $(shell id -g)
+DOCKER_USER = $(DOCKER_UID):$(DOCKER_GID)
+COMPOSE = DOCKER_USER=$(DOCKER_USER) docker compose
+COMPOSE_RUN = $(COMPOSE) run --rm
+COMPOSE_RUN_HUGO = $(COMPOSE_RUN) hugo
+
+# ==============================================================================
+# RULES
+
+default: help
+
+# -- Docker
+run: ## run dev server
+	@$(COMPOSE) up --force-recreate -d hugo
+.PHONY: run
+
+stop: ## stop container
+	@$(COMPOSE) stop
+.PHONY: stop
+
+down: ## stop and remove container
+	@$(COMPOSE) down
+.PHONY: down
+
+logs: ## display hugo logs
+	@$(COMPOSE) logs -f hugo
+.PHONY: logs
+
+status: ## an alias for "docker-compose ps"
+	@$(COMPOSE) ps
+.PHONY: status
+
+## -- Git
+git-modules-init: ## Init git modules
+	 git submodule update --init --recursive
+.PHONY: gitmodules-init
+
+git-modules-status: ## Display git modules status (e.g: to display version)
+	git submodule status
+.PHONY: gitmodules-status
+
+git-clean: ## restore repository state as it was freshly cloned
+	git clean -idx
+.PHONY: clean
+
+# -- Project
+bootstrap: ## Bootstrap project
+bootstrap: \
+	gitmodules-init \
+	run
+.PHONY: bootstrap
+
+build: ## Build static blog
+	@$(COMPOSE_RUN_HUGO) -t $(THEME)
+.PHONY: build
+
+update-theme: ## Pull the theme submodules from its origin
+	cd ./themes/$(THEME)
+	git fetch origin
+	git pull origin --rebase --autostash
+.PHONY: update-theme
+
+# -- Articles
+new-post: ## Create a new post
+	bin/new-post
+.PHONY: new-post
```

Typing `new` dash `post` is not comfortable, what about the following?

```suggestion
new: ## Create a new post
	bin/new-post
.PHONY: new
```

---

I suggest to use post instead. In this way, if we want later add commands to add author and so on, we will not have collisions.